packages/components/Organisms/HeroBanner/HeroBanner.vue

```diff
@@ -160,39 +160,66 @@ export default {
 
   $rpl-hero-banner-left-padding: rem(60px) - $rpl-component-gutter-l;
 
+  // Background graphic positioning - Set positioning for a breakpoint:
+  //  - pos: 'edge / right' - Edge starts graphic at right screen edge.
+  //  - size: The graphic size. This matches the top layout graphic.
+  //  - gutter: Edge only. Defines space between container & screen edge.
+  //  - v-offset: Edge & Right. Additional vertical offset to place on graphic.
+  //  - h-offset: Edge only. Additional horizontal offset to place on graphic.
+  $rpl-hero-banner-background-graphic-links-overlap: rem(100px) !default;
+  $rpl-hero-banner-background-graphic-gutter-xl: '((100vw - #{map-get(map-get($rpl-layout, 'site_max_width'), 'xl')})/2)';
+  $rpl-hero-banner-background-graphic-v-offset-xxl: rem(-68px);
+  $rpl-hero-banner-background-graphic: (
+    'xs': ('pos': 'right', 'size': 18rem, 'v-offset': ($rpl-space * -5)),
+    's': ('pos': 'edge', 'size': 20rem, 'gutter': $rpl-component-padding-xs, 'v-offset': rem(-16px)),
+    'm': ('pos': 'edge', 'gutter': map-get(map-get($rpl-layout, 'site_padding'), 'm')),
+    'l': ('pos': 'edge', 'gutter': map-get(map-get($rpl-layout, 'site_padding'), 'l')),
+    'xl': ('pos': 'edge', 'gutter': $rpl-hero-banner-background-graphic-gutter-xl),
+    '1490px': ('pos': 'right'),
+    'xxl': ('pos': 'edge', 'size': 40rem, 'gutter': $rpl-hero-banner-background-graphic-gutter-xl, 'h-offset': rem(33px), 'v-offset': $rpl-hero-banner-background-graphic-v-offset-xxl),
+    '2072px': ('pos': 'right', 'v-offset': $rpl-hero-banner-background-graphic-v-offset-xxl),
+    'xxxl': ('pos': 'right', 'v-offset': $rpl-hero-banner-background-graphic-v-offset-xxl),
+    '2660px': ('pos': 'right')
+  ) !default;
```

Comment:
This is really a complex code solution for a design issue. I would suggest review the design instead of adding complex logic into component as it will be hard to add new variations in the future. I have checked the result http://storybook-ripple-pr-478.lagoon.vicsdp.amazee.io/?path=/story/organisms-herobanner--default which is working well. I will pass it, cheers!